README.md

```diff
@@ -370,6 +395,7 @@ There are several options that are supported on the main app and subcommands. Th
 -   `.set_help_all_flag(name, message)`: Set the help all flag name and message, returns a pointer to the created option. Expands subcommands.
 -   `.failure_message(func)`: Set the failure message function. Two provided: `CLI::FailureMessage::help` and `CLI::FailureMessage::simple` (the default).
 -   `.group(name)`: Set a group name, defaults to `"Subcommands"`. Setting `""` will be hide the subcommand.
+- `[option_name]`: retrieve a const reference to an option given by `option_name` for Example `app["--flag1"]` will get an option reference for the "--flag1" value,  `app["--flag1"].as<bool>() will get the results of the command line for a flag
```

Comment:
If we make sure options cannot be copied (copy would need to be a protected function with some friends, I believe), this might be okay since it is intended to be chained. This should at least fail to compile:

```cpp
auto flag1 = app["--flag1"];
```

Reply:
I moved it to a pointer,  copying an option would probably not be the intended result.

Reply:
Okay, I like that, and it is consistent. Probably should verify options can't be accidentally copied (not in this PR).